### PR TITLE
Lixulia-Case

### PR DESCRIPTION
**Pull request by ZRock35**
Adding Arcturus case to Cases section, also images.

### CODE REVIEW

File: diy/cases.md [hunk 1]
```diff
@@ -125,4 +125,18 @@ The SlimeVR community has built a huge number of cases for style, internals and
...
+
+![Case](https://github.com/Lixulia/Arcturus/blob/main/renderforarcturus.PNG) {: .caseImage}
+
+* D1 Mini ESP microcontroller
+* DPDT 2P2T Power Switch
+* BMI160 
+* TP4056 USB-C charging module
+* 804040 1200mAh lithium ion cell
+
+[Github](https://github.com/Lixulia/Arcturus) (edited)
```
Comment (Louka3000) [inline diy/cases.md:139]:
```suggestion
[Github](https://github.com/Lixulia/Arcturus)
```

File: diy/cases.md [hunk 2]
```diff
@@ -125,4 +125,18 @@ The SlimeVR community has built a huge number of cases for style, internals and
 
 [Github](https://github.com/TwidgeVR/slimevr_compact_case)
 
+## Arcturus by Lixulia
+Design by Lixulia
+
+![Case](https://github.com/Lixulia/Arcturus/blob/main/renderforarcturus.PNG) {: .caseImage}
```
Comment (Louka3000) [inline diy/cases.md:131]:
```suggestion
![Case](https://raw.githubusercontent.com/Lixulia/Arcturus/main/renders/renderforarcturus.PNG) {: .caseImage}
```

File: diy/cases.md (hunk 1 above)
Comment (Louka3000) [inline diy/cases.md]:
```suggestion
[Github](https://github.com/Lixulia/Arcturus)
```

**Review by Louka3000 — CHANGES_REQUESTED** [review]

File: diy/cases.md (hunk 1 above)
Comment (Louka3000) [reply diy/cases.md]: oops duplicate

**Review by Louka3000 — APPROVED** [review]

**Review by calliePepper — APPROVED** [review]

**Pull request merged**